exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -88,6 +91,26 @@ public ZipkinSpanExporterBuilder setEndpoint(String endpoint) {
     return this;
   }
 
+  /**
+   * Sets the method used to compress payloads. If unset, compression is enabled. Currently
+   * supported compression methods include "gzip" and "none".
+   *
+   * <p>The compression method is ignored when a custom Zipkin sender is set via {@link
+   * #setSender(Sender)}.
+   *
```

Comment:
nit: we should add a note about the default value in the builder javadoc as well.

Reply:
Could you clarify what you mean by this? This method's javadoc already includes "If unset, compression is enabled.".

Reply:
I guess I would explicitly call out that "gzip" is the default value, if unset, rather than leaving it vague-ish. Not a big deal if you think this is clear enough.

Reply:
Ah ok. Yeah that makes sense. The wording was lifted from other exporters where the default is disabled, so the phrasing "If unset, compression is disabled." is unambiguous. When the default is compression, makes sense to specify which compression algo.